app/views/catalog/_universal_viewer_default.html.erb

```diff
@@ -1,6 +1,5 @@
 <% if resource.class.can_have_manifests? && resource.try(:member_ids).present? %>
   <div class="intrinsic-container intrinsic-container-16x9 uv-container">
-    <%# note: UV html file is '/uv/uv' %>
```

Comment:
Very minor thing: would you mind copying the comment to the new location of the UV path — I agree with @hackmastera's comment that the duplicated `/uv` might look like a mistake, so a comment clarifying it's correct is a good idea.